js/challenges/002_MergeTwoSortedArrays/index.mjs

```diff
@@ -0,0 +1,23 @@
+/* 
+ * Returns a new sorted array with every item from each of two input sorted arrays.
+ * @param {array} arr1 Input sorted array
+ * @param {array} arr2 Input sorted array
+ * @returns {array} New combined sorted array
+ */
+export function mergeTwoSortedArrays(arr1, arr2) {
+    var newArray = [];
+
+    while((arr1.length > 0) && (arr2.length > 0)) {
+        if (arr1[0] < arr2[0]) {
+            newArray.push(arr1.shift());
```

Comment:
`shift` is an expensive call to make, it is a O(n) operation, meaning that if the array has 500 elements, the shift results in 499 operations.

try a different approach that is less expensive.